src/db/gc/snapshot.rs

```diff
@@ -239,6 +249,30 @@ where
                         }
                     }
                 }
+
+                // Backfill new db records during snapshot export
+                if let Some(mem_db) = self.memory_db.write().take() {
+                    let count = mem_db.len();
+                    let approximate_heap_size = {
+                        let mut s = 0;
+                        for (_k, v) in mem_db.iter() {
+                            s += 64;
+                            s += v.len();
+                        }
+                        s
+                    };
+                    if let Ok(db) = open_db(self.db_root_dir.clone(), &self.db_config) {
+                        let start = Instant::now();
+                        if let Err(e) = db.put_many_keyed(mem_db) {
+                            tracing::warn!("{e}");
+                        }
+                        tracing::info!(
```

Comment:
Sample log from my laptop on `mainnet`:
```
2025-05-12T09:19:35.957856Z  INFO forest::db::gc::snapshot: backfilled 57345 new db records since snapshot epoch, approximate heap size: 82.9 MiB, took 559ms 304us 167ns
```